src/routes/favorites/FavoritesScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import { RouteProp, useFocusEffect } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React, { ReactElement } from 'react'
+import { FlatList } from 'react-native'
+import styled from 'styled-components/native'
+
+import RouteWrapper from '../../components/RouteWrapper'
+import ServerResponseHandler from '../../components/ServerResponseHandler'
+import Title from '../../components/Title'
+import { Favorite } from '../../constants/data'
+import { Document } from '../../constants/endpoints'
+import useLoadAsync from '../../hooks/useLoadAsync'
+import { RoutesParams } from '../../navigation/NavigationTypes'
+import { getFavorites } from '../../services/AsyncStorage'
+import { getLabels } from '../../services/helpers'
+import FavoriteItem from './components/FavoriteItem'
+
+interface FavoritesScreenProps {
+  route: RouteProp<RoutesParams, 'Favorites'>
+  navigation: StackNavigationProp<RoutesParams, 'Favorites'>
+}
+
+const Root = styled.View`
+  margin: 10px;
```

Comment:
we should stick to our spacings or at least use `hp(xy%)` . Even you didn't change it now it would nice to fix it :)